ddtrace/settings/config.py

```diff
@@ -198,10 +198,10 @@ def get_error_ranges(error_range_str):
 class _ConfigItem:
     """Configuration item that tracks the value of a setting, and where it came from."""
 
-    def __init__(self, name, default, envs):
-        # type: (str, Union[_JSONType, Callable[[], _JSONType]], List[Tuple[str, Callable[[str], Any]]]) -> None
+    def __init__(self, default, envs):
+        # type: (Union[_JSONType, Callable[[], _JSONType]], List[Tuple[str, Callable[[str], Any]]]) -> None
         # _ConfigItem._name is only used in __repr__ and instrumentation telemetry
-        self._name = name
+        self._name = envs[0][0]
```

Comment:
>  The name of a configuration will be set to the first environment variable in the envs field (ie the primary environment variable).

Is this always guaranteed? Wouldn't it be better to always just have it passed in directly like before?

Reply:
Yup but this felt redundant. Also this could result in some configurations having names that are not equal to the environment variable. In my opinion all configurations should always be named after their primary environment variable. This will improve consistency across languages. We should avoid arbitrary names as much as possible.